packaging/rhc-worker-bash.spec

```
@@ -18,13 +18,13 @@
 %endif
 
 Name:           %{repo_name}
-Version:        0.1
+Version:        0.2
 Release:        1%{?dist}
 Summary:        Remote Host Configuration (rhc) worker for executing bash scripts on hosts managed by Red Hat Insights.
 
 License:        GPLv3+
 URL:            https://github.com/%{repo_orgname}/%{repo_name}/
```

The trailing slash is not needed - but I guess it shouldn't break anything?

```suggestion
URL:            https://github.com/%{repo_orgname}/%{repo_name}
```

---

Probably would break things, good catch!